behave_test_automation/pah_behave/features/new_drive_page.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Add new drive view
+  In order to be able to add a new drive
+  As a user
+  I want to submit new drive
+
+  Scenario: User sees field validation errors on add new drive view
+    Given User logs in to pah-fm system
+    When User submits empty add new drive form
+    Then User sees an errors list
+
+  Scenario Outline: User submits a new drive
+    Given User logs in to pah-fm system
+    When User submits "<start location>", "<starting mileage>", "<end location>" and "<ending mileage>"
```

Comment:
There is some inconsistency with naming columns/variables. Sometimes it's `starting_mileage & ending_mileage`, sometimes it's `starting_mileage & end_mileage`. If you don't mind - please change all of them to one form (I'd prefer the `start_mileage & end_mileage` form).

Reply:
Changes applied. Changed to "start_mileage" and "end_mileage" as requested :)